Mathlib/Algebra/CharP/Algebra.lean

```diff
@@ -30,30 +30,33 @@ Instances constructed from this result:
 -/
 
 
+/-- If a ring homomorphism `R →+* A` is injective then `A` has the same characteristic as `R`. -/
+theorem charP_of_injective_ringHom {R A : Type*} [CommSemiring R] [Semiring A] {f : R →+* A}
```

Comment:
Does R need to be commutative? (three occurrences total)

Reply:
No 😂